target/linux/ath79/dts/ar9132_buffalo_wzr-hp-g300nh-rb.dts

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "ar9132_buffalo_wzr-hp-g300nh.dtsi"
+
+&{/} {
+	compatible = "buffalo,wzr-hp-g300nh-rb", "qca,ar9132";
+};
+
+&switch {
+	status = "okay";
```

Comment:
Is there a reason to have the statusses in DTS though they are set to the same value?

Reply:
in the .dtsi the switch is set to status "disabled". I assumed it's a common convention to mark devices disabled until fully described. On the other hand, the mdio-bus subnode is being set to status = "okay" so one could argue that there's a discrepancy there. I have no strong opinion either way, I can change it/them the way is preferred here.

Reply:
Well, since the DTSI does not really describe a working configuration anyway, I'd personally have moved stuff there which is the same in both DTSes. But I also don't consider this high priority.

Reply:
Oh but the common stuff is in the DTSI file, the DTS files only contains what's different between the devices. Or am I missing something here...?

Reply:
status=okay is also common :-)

Reply:
Oh you mean that. :) My thinking behind it was that because the compatible-property needs to be set it's not ready and "ok" and thus the DTSI has them "disabled" and only once the compatible has been set it's "ok". If I'll leave as it is now so it'll spare me from smoke testing.